### DISCUSSION TITLE
M2O O2M Corresponding Field Default Field Name

### DISCUSSION THREAD

**Discussion by bherbruck** [directus/directus#3292] [Feature Requests]
When you create an O2M Corresponding field from the M2O field editor, it defaults the field name to the 'one' side of the relationship instead of the 'many' side. It would be nice if it would default to the 'many' side (expressed in plural form if possible).

In the example of the screenshot below, it currently defaults to creating customer.customer instead of customer.order[s].

![Screenshot 2020-12-03 135656](https://user-images.githubusercontent.com/5258126/101075692-5abee000-3570-11eb-996c-5b54ff3ac689.png)

**Comment by Michael-Nussbaumer** [comment]
@bherbruck open an Issue about that, think that is not intended. When you create a M2M it uses the correct preset Field Name

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
